src/components/TweetsContainer/index.js

```diff
@@ -175,6 +176,7 @@ function TweetsContainer({
       />
       {isLoading && <Loading />}
       <Chart {...props} data={insights} classes={{ root: classes.chartRoot }} />
+      <SearchResults results={query} label="Search Results" />
```

Comment:
```suggestion
      <SearchResults query={query} label="Search Results" />
```
I think `query` or `keyword` is better than `results` here.